packages/calypso-codemods/transforms/sort-imports.js

```diff
@@ -112,7 +112,7 @@ module.exports = function ( file, api ) {
 		externalDependenciesSet.has( importNode.source.value.split( '/' )[ 0 ] );
 
 	// if there are no deps at all, then return early.
-	if ( _.isEmpty( declarations.nodes() ) ) {
+	if ( ! declarations.nodes() || declarations.nodes().length === 0 ) {
```

Comment:
Nit: Should we assign `declarations.nodes()` to a constant so it's invoked just once?

Reply:
Yes. done